modyn/supervisor/internal/triggers/utils/model/manager.py

```diff
@@ -11,10 +11,10 @@
 logger = logging.getLogger(__name__)
 
 
-class EmbeddingEncoder:
-    """The EmbeddingEncoder stores a model and its metadata.
+class ModelManager:
```

Comment:
can we find a better term than manager? it's actually considered to be a bit of a code smell to call something manager. maybe there is a gang of four pattern that matches this?

Reply:
`Wrapper`? `EmbeddingEncoder` was definitely off, as the class doesn't have anything to do with either embeddings or encoding😅